Assets/Scripts/Gamemode/Racing/RacingObjective.cs

```diff
@@ -77,8 +77,8 @@ private void Start()
                 Destroy(gameObject.transform.parent.gameObject);
                 return;
             }
-            Gamemode.Objectives.Add(this);
-            State = ObjectiveState.Queue;
+	        State = ObjectiveState.Queue;
+	        Gamemode.AddObjective(this);
```

Comment:
Any particular reason these are swapped?

Reply:
Thanks for clarifying!

Reply:
I do agree that it is a bit odd, the Objectives are not found by the RacingGamemode script, they register themselves to RacingGamemode, I didn't change that to not alter the code to much.